src/Integrations/Integrations/Symfony/SymfonyIntegration.php

```diff
@@ -60,16 +55,31 @@ function (SpanData $span) {
         \DDTrace\trace_method(
             'Symfony\Component\HttpKernel\Kernel',
             'boot',
-            function (SpanData $span) {
-                $span->name = 'symfony.httpkernel.kernel.boot';
-                $span->resource = \get_class($this);
-                $span->type = Type::WEB_SERVLET;
-                $span->service = \ddtrace_config_app_name('symfony');
-            }
+            [
+                "prehook" => function () {
+                    if ($rootSpan = \DDTrace\root_span()) {
+                        $this->appName = \ddtrace_config_app_name('symfony');
+                        $rootSpan->name = 'symfony.request';
+                        $rootSpan->service = $this->appName;
+                    }
+                },
+                "posthook" => function (SpanData $span) {
+                    $span->name = 'symfony.httpkernel.kernel.boot';
+                    $span->resource = \get_class($this);
+                    $span->type = Type::WEB_SERVLET;
+                    $span->service = \ddtrace_config_app_name('symfony');
+                }
+            ]
```

Comment:
For this one, can't we just do everything in the prehook section?

Reply:
We can. Changed.